docs/insomnia/get-started-with-documents.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: article-detail
+title: Get Started with Documents
+category: "API Design"
+category-url: api-design
+---
+
+Design Documents (also called Documents) hold specifications, API requests, and tests. They also offer the capability to sync with Git.
+
+If you're looking for information about Collections instead (which use Version Control Sync), see [Request Collections](/insomnia/request-collections).
```

Comment:
should we link to Version Control Sync too? Or is that too many links?

Reply:
I can link it. I think it's not too many links